<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VTKRender2DWidget</class>
 <widget class="QWidget" name="VTKRender2DWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QVTKOpenGLNativeWidget" name="openGLWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="m_sliceIndexLb">
        <property name="text">
         <string>Slice:</string>
        </property>
        <property name="VTKRender2DWidgetLabel" stdset="0">
         <string>PatientInfo</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="m_patientInfoPanel">
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="m_patientIDLbN">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>病人ID：</string>
           </property>
           <property name="VTKRender2DWidgetLabel" stdset="0">
            <string>PatientInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_patientNameLbN">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>姓名：</string>
           </property>
           <property name="VTKRender2DWidgetLabel" stdset="0">
            <string>PatientInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_patientSexLbN">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>性别：</string>
           </property>
           <property name="VTKRender2DWidgetLabel" stdset="0">
            <string>PatientInfo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="m_patientAgeLbN">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="text">
            <string>年龄：</string>
           </property>
           <property name="VTKRender2DWidgetLabel" stdset="0">
            <string>PatientInfo</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1052</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" rowspan="2">
       <widget class="QSlider" name="horizontalScrollBar">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>706</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>qvtkopenglnativewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
